plugin/vcs/gitlab/sql-review.yml

```diff
@@ -0,0 +1,23 @@
+sql-review:
+  only:
+    refs:
+      - merge_requests
+  image: docker:stable
+  variables:
+    API: "%s" # This is the API for SQL reivew.
+  before_script:
+    - apk update && apk add curl
+    - apk update && apk add jq
+  script:
+    - echo "Start request $API/pull/$CI_MERGE_REQUEST_IID"
+    - 'response=$(curl -s --show-error -X GET "$API/pull/$CI_MERGE_REQUEST_IID" -H "Content-type: application/json" -H "X-SQL-Review-Token: $%s")'
```

Comment:
The YML is updated. We just simply call the SQL review endpoint and output the response now. The complex logic is moved to https://github.com/bytebase/bytebase/pull/2862
PTAL @d-bytebase